### PR TITLE
feat: section about Enums, file naming convention

### PR DESCRIPTION
**Pull request by igorwessel**
I've added section about Enums and file naming convention for files that are not components.

# What was added?

Folder Architecture
2.2 Scoped Files

React
7.7 Enums

### CODE REVIEW

**Review by RodrigoRVSN — DISMISSED** [review]

**Review by felipefialho — APPROVED** [review]
🙏

File: README.md [hunk 1]
```diff
@@ -172,6 +172,25 @@ For example:
...
+┣ 📂 pages \
+┣ ┣ 📂 Home \
+┣ ┃ ┣ 📂 utils \
+┣ ┃ ┣ ┣ 📜 someUtils.js
+┣ ┃ ┣ 📂 helpers \
+┣ ┣ ┣ ┣ 📜 someHelper.js
+┣ ┃ ┣ 📂 hooks \
+┣ ┣ ┣ ┣ 📜 useSomeHook.js
+```
+
```
Comment (munyrasirio) [inline README.md:193]: Suggestion: This example is for the first scenario, maybe we could put it right after the description of it. Markdown got messed up, but I'll try to show what I'm thinking:

```
We need to add inside `pages/**/{utils, helpers, context, hooks, etc...}` and use `camelCase` as **Naming Convention**.

[your exemple here]

If we need to used this files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

File: README.md [hunk 2]
```diff
@@ -172,6 +172,25 @@ For example:
 ┣ ┣ ┃ ┃ ┣ 📜 UserProfile.stories.mdx \
 ```
 
+#### Scoped Files 
+
+We need to add inside `pages/**/{utils, helpers, context, hooks, etc...}` and use `camelCase` as **Naming Convention**.
+
+If we need to used this files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```
Comment (munyrasirio) [inline README.md:179]:
```suggestion
If we need to use these files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

File: README.md [hunk 3]
```diff
@@ -936,6 +956,43 @@ const MyComponent = () => (
 )
 ```
 
+### 7.7 Avoid compare directly strings
+
+When know all possible values we can use enum to achieve better readability, control.
```
Comment (munyrasirio) [inline README.md:961]:
```suggestion
When know all possible values we can use enum to achieve better readability and control.
```

Comment (igorwessel) [reply README.md]: thanks!! fix in [91eaf3b](https://github.com/juntossomosmais/frontend-guideline/pull/29/commits/91eaf3b5bc124ec841b837a8113ac43c04012258)

File: README.md (hunk 2 above)
Comment (igorwessel) [reply README.md]: thanks!! fix in [91eaf3b](https://github.com/juntossomosmais/frontend-guideline/pull/29/commits/91eaf3b5bc124ec841b837a8113ac43c04012258)

File: README.md (hunk 1 above)
Comment (igorwessel) [reply README.md]: Nice suggestion!! I created the examples for when moving these guys to a global context :)
thanks, done in [f642d5f](https://github.com/juntossomosmais/frontend-guideline/pull/29/commits/f642d5f942236ce61c1602bd8b1f76ecaa25d364)

File: README.md [hunk 4]
```diff
@@ -172,6 +172,35 @@ For example:
...
+┣ ┣ 📂 Home \
+┣ ┃ ┣ 📂 utils \
+┣ ┃ ┣ ┣ 📜 someUtils.js
+┣ ┃ ┣ 📂 helpers \
+┣ ┣ ┣ ┣ 📜 someHelper.js
+┣ ┃ ┣ 📂 hooks \
+┣ ┣ ┣ ┣ 📜 useSomeHook.js
+```
+
+If we need use these this files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```
Comment (munyrasirio) [inline README.md:192]:
```suggestion
If we need use these files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

"this" is still there. 😆 
[_suspense song intensifies_]

File: README.md [hunk 5]
```diff
@@ -172,6 +172,35 @@ For example:
...
+
+```sh
+┣ 📂 src
+┣ ┣ 📂 utils \
+┣ ┃ ┣ 📜 someUtils.js
+┣ ┣ 📂 helpers \
+┣ ┃ ┣ 📜 someHelper.js
+┣ ┣ 📂 hooks \
+┣ ┃ ┣ 📜 useSomeHook.js
+```
```
Comment (munyrasirio) [inline README.md:202]: Praise: Nice!

File: README.md (hunk 4 above)
Comment (igorwessel) [reply README.md:192]: :hehe:

Comment (igorwessel) [reply README.md]: done :sai-pra-la-bug:

**Review by munyrasirio — APPROVED** [review]
🚀

**Review by felipefialho — APPROVED** [review]

**Review by RodrigoRVSN — APPROVED** [review]

**Pull request merged**